CS3219-AY2425S1/cs3219-ay2425s1-project-g11
# Feat/code execution

Add Code Execution Functionality

Main changes:
- Fix syntax highlighting bug
- Extended syntax highlighting support for multiple languages
- Added Code execution

To Test:
- same old procedure to set it up
- go to the coding page
- type in arbitrary hello world codes
- click on the run button
- a card with the output will appear
- click anywhere outside of the card to hide the card
